### PR TITLE
changed: remove .rar|.cbr from default video/music/image extensions

### PR DESCRIPTION
**Pull request by notspiff**
these are added if the add-ons are enabled.

### CODE REVIEW

File: xbmc/settings/AdvancedSettings.cpp [hunk 1]
```diff
@@ -380,9 +380,9 @@ void CAdvancedSettings::Initialize()
...
+  m_pictureExtensions = ".png|.jpg|.jpeg|.bmp|.gif|.ico|.tif|.tiff|.tga|.pcx|.cbz|.zip|.rss|.webp|.jp2|.apng";
+  m_musicExtensions = ".nsv|.m4a|.flac|.aac|.strm|.pls|.rm|.rma|.mpa|.wav|.wma|.ogg|.mp3|.mp2|.m3u|.gdm|.imf|.m15|.sfx|.uni|.ac3|.dts|.cue|.aif|.aiff|.wpl|.ape|.mac|.mpc|.mp+|.mpp|.shn|.zip|.wv|.dsp|.xsp|.xwav|.waa|.wvs|.wam|.gcm|.idsp|.mpdsp|.mss|.spt|.rsd|.sap|.cmc|.cmr|.dmc|.mpt|.mpd|.rmt|.tmc|.tm8|.tm2|.oga|.url|.pxml|.tta|.rss|.wtv|.mka|.tak|.opus|.dff|.dsf|.m4b";
+  m_videoExtensions = ".m4v|.3g2|.3gp|.nsv|.tp|.ts|.ty|.strm|.pls|.rm|.rmvb|.mpd|.m3u|.m3u8|.ifo|.mov|.qt|.divx|.xvid|.bivx|.vob|.nrg|.img|.iso|.udf|.pva|.wmv|.asf|.asx|.ogm|.m2v|.avi|.bin|.dat|.mpg|.mpeg|.mp4|.mkv|.mk3d|.avc|.vp3|.svq3|.nuv|.viv|.dv|.fli|.flv|.001|.wpl|.zip|.vdr|.dvr-ms|.xsp|.mts|.m2t|.m2ts|.evo|.ogv|.sdp|.avs|.rec|.url|.pxml|.vc1|.h264|.rcv|.rss|.mpls|.webm|.bdmv|.wtv|.trp|.f4v";
   m_subtitlesExtensions = ".utf|.utf8|.utf-8|.sub|.srt|.smi|.rt|.txt|.ssa|.text|.ssa|.aqt|.jss|.ass|.idx|.ifo|.rar|.zip";
```
Comment (Rechi) [inline xbmc/settings/AdvancedSettings.cpp]: can `.rar` also be removed here?

Comment (notspiff) [reply xbmc/settings/AdvancedSettings.cpp]: no, there is no FileExtensionProvider handling for these.

Comment (Rechi) [reply xbmc/settings/AdvancedSettings.cpp]: something missing at the following code?
https://github.com/xbmc/xbmc/blob/adb2c3529584d28b3c83feb46576620a83130d0d/xbmc/utils/FileExtensionProvider.cpp#L80-L86

Comment (notspiff) [reply xbmc/settings/AdvancedSettings.cpp]: nope, just me who didn't spot it.

**Comment by MartijnKaijser** [comment]
Please also check both manifest for windows if they include the same extensions

**Comment by notspiff** [comment]
we ship the add-on(s) so i don't see why that should change.

**Comment by MartijnKaijser** [comment]
Jenkins build this please

**Pull request merged**